### PR TITLE
cnwan-operator v1.0.0 chart

### PR DESCRIPTION
**Pull request by SunSince90**
This PR adds all relevant files for version `1.0.0` of the helm chart for the cnwan-operator, including `Chart.yaml`, `values.yaml` and all other files inside `/templates` and workflows for auto-releasing on merge and generating chart readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,84 @@
...
+
+Helm helps you manage Kubernetes applications — *Helm Charts* help you define,
+install, and upgrade even the most complex Kubernetes application.
+
+To install Helm, refer to the [Helm install guide](https://github.com/helm/helm#install)
+and ensure that the `helm` binary is in the `PATH` of your shell.
+
+### Add Repo
+
+Run this comman to download -- and later install -- all the charts from this
```
Comment (ljakab) [inline README.md:31]: s/comman/command/

File: README.md [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+on this command:
+
+```bash
+helm install cnwan-operator cnwan/cnwan-operator <settings>
+```
+
+Please refer to *CN-WAN Operator*'s [chart guide](./cnwan-operator/README.md)
+to learn more about the installation process and its
+[official repository](https://github.com/CloudNativeSDWAN/cnwan-operator) for
+the settings and understand how it works.
```
Comment (ljakab) [inline README.md:52]: s/understand/understanding/

File: README.md [hunk 3]
```diff
@@ -1,3 +1,84 @@
...
+    `git checkout -b my-new-feature`
+4. Make changes and add them
+    `git add .`
+5. Commit your changes
+    `git commit -m 'Add some feature'`
+6. Push to the branch
+    `git push origin my-new-feature`
+7. Create new pull request to this repository
+
+Before opening a pull request, please make sure you have update any relevant
```
Comment (ljakab) [inline README.md:72]: s/update/updated/ (or remove the word "have")

File: README.md [hunk 4]
```diff
@@ -1,3 +1,84 @@
...
+values in `Chart.yaml` and `values.yaml`, including:
+
+- `appVersion` in `Chart.yaml`
+- `version` in `Chart.yaml`
+- `description` in `Chart.yaml`
+- `image.tag` in `values.yaml`
+
+and document any changes that you have made and would like to be merged.
```
Comment (ljakab) [inline README.md:80]: s/document/documented/

File: charts/cnwan-operator/README.md.gotmpl [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+{{ template "chart.valuesTable" . }}
+
+***
+
+## Examples
+
+The following are quickstarts/examples to demonstrate how to deploy the
+operator and configure it to use a certain service registry.
+
```
Comment (ljakab) [inline charts/cnwan-operator/README.md.gotmpl:51]: I would also add a quick section here to mention that in addition to using `--set` on the CLI, user can also create a custom `values.yaml` file of their own with their settings, which is more readable, especially if they want to override/set a lot of values.

File: charts/cnwan-operator/values.yaml [hunk 6]
```diff
@@ -0,0 +1,127 @@
+image:
+  # image.registry -- The registry where to get the container image.
+  registry: ghcr.io
+  # image.repository -- The repository where to get the container image inside
+  # the registry specified above.
+  repository: cloudnativesdwan/cnwan-operator
+  # image.tag -- The tag for the container image.
+  # @default -- The most recent version will be used.
+  tag: v0.5.0
```
Comment (ljakab) [inline charts/cnwan-operator/values.yaml:9]: We should change this `latest`? At least that's what the `@default` hints at

**Review by ljakab — CHANGES_REQUESTED** [review]
Just a few minor comments inline. We also need a `NOTICE` file with the copyright, like in the other repos.

File: charts/cnwan-operator/values.yaml (hunk 6 above)
Comment (SunSince90) [reply charts/cnwan-operator/values.yaml]: Resolving as already discussed on WebEx.

File: charts/cnwan-operator/README.md.gotmpl (hunk 5 above)
Comment (SunSince90) [reply charts/cnwan-operator/README.md.gotmpl:64]: Done, lmk if it's better

**Review by ljakab — APPROVED** [review]
Great, thanks!

**Pull request merged**